src/Paynow/HttpClient/HttpClient.php

```diff
@@ -4,21 +4,22 @@
 
 use Http\Client\Curl\Client;
 use Http\Client\Exception\RequestException;
-use Http\Message\MessageFactory\GuzzleMessageFactory;
-use Http\Message\StreamFactory\GuzzleStreamFactory;
+use Http\Discovery\Psr17FactoryDiscovery;
 use Paynow\Configuration;
 use Paynow\Util\SignatureCalculator;
 use Psr\Http\Message\RequestInterface;
 
 class HttpClient implements HttpClientInterface
 {
-    /**
-     * @var Client
-     */
+    /** @var Client */
     protected $client;
 
+    /** @var \Psr\Http\Message\RequestFactoryInterface  */
     protected $messageFactory;
 
+    /** @var \Psr\Http\Message\StreamFactoryInterface  */
```

Comment:
you can easily simplify a FQN here in exactly the same nammer which is done for `$client` property

Reply:
Done